src/Umbraco.Cms.Integrations.Search.Algolia/Api/Management/Controllers/SaveIndexController.cs

```diff
@@ -56,11 +58,12 @@ public async Task<IActionResult> SaveIndex([FromBody] IndexConfiguration index)
                     ? Result.Ok()
                     : await IndexService.PushData(index.Name);
 
-                return new JsonResult(result);
+                return Ok(result);
             }
             catch (Exception ex)
             {
-                return new JsonResult(Result.Fail(ex.Message));
+                Logger.LogError(ex, ex.Message);
```

Comment:
Wondering if you really need the `try/catch`?  As I think any Umbraco exception will get logged anyway.

Reply:
Confirmed, Umbraco will log the exceptions.